streams/src/main/java/org/apache/kafka/streams/kstream/internals/KTableImpl.java

```diff
@@ -137,7 +137,7 @@ public void print() {
     @Override
     public void print(Serde<K> keySerde, Serde<V> valSerde) {
         String name = topology.newName(PRINTING_NAME);
-        topology.addProcessor(name, new KeyValuePrinter<>(keySerde, valSerde), this.name);
+        topology.addProcessor(name, new KeyValuePrinter<>(keySerde, valSerde, this.name), this.name);
```

Comment:
With that in mind, I'll add the same overloaded methods for `print()` that were provided in `KStream`.